docs/platforms/python/index.mdx

```diff
@@ -19,6 +19,12 @@ Don't already have an account and Sentry project established? Head over to [sent
 
 If you prefer to follow video instructions, see [How to Install the Sentry Python SDK in 60 Seconds](https://vimeo.com/899368672).
```

Comment:
Note to double check this video after version 2 is released to see if it needs any updates or if this line should be removed. I'm not sure who made the video though...

Reply:
Afaik the video was made by Salma -- but we should be good to leave it as is, everything still works the same way from the user point of view. Just watched it to double check.